src/core/Akka/Event/EventBusUnsubscriber.cs

```diff
@@ -90,7 +90,7 @@ protected override void PreStart()
         ///
         /// Registers a new subscriber to be death-watched and automatically unsubscribed.
         /// </summary>
-        internal class Register
+        internal class Register : INoSerializationVerificationNeeded
```

Comment:
Is this the intended design for this class? Classes that implement `INoSerializationVerificationNeeded` won't be serialized if configuration setting 'akka.actor.serialize-messages' is "on"

Reply:
Yes, this is entirely internal and is `IActorRef` is immutable,

Reply:
Got it!